### PR TITLE
veda sandbox

### PR DESCRIPTION
**Pull request by muhammad-ammar**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

**Comment by muhammad-ammar** [comment]
@feanil These are the changes required for sandbox provisioning. Please have a look.

**Comment by muhammad-ammar** [comment]
[Sandbox](https://pipeline-ma101.sandbox.edx.org) with `full_edx_installation` 
[Sandbox](https://pipeline-ma102.sandbox.edx.org) with `full_edx_installation_from_scratch`

File: playbooks/roles/edxapp/tasks/deploy.yml [hunk 1]
```diff
@@ -437,6 +437,17 @@
     - manage
     - manage:db
 
+- name: create edxapp video pipeline integration
+  shell: "{{ edxapp_venv_bin }}/python ./manage.py lms --settings={{ edxapp_settings }} create_video_pipeline_integration {{ item.client_name}} {{ item.api_url }} {{ item.service_username }} --enabled"
```
Comment (feanil) [inline playbooks/roles/edxapp/tasks/deploy.yml]: What is the goal of this, is this? What does it configure?  Has this ever been run in production?

File: playbooks/roles/veda_ffmpeg/defaults/main.yml [hunk 2]
```diff
@@ -13,6 +13,6 @@
 
 VEDA_FFMPEG_GIT_URL: 'https://github.com/yro/v_videocompile'
 # used /opt path to clone the repo and then install ffmpeg using the cloned repo
-veda_ffmpeg_root: '/opt/veda_ffmpeg'
+veda_ffmpeg_root: '/var/tmp/veda_ffmpeg'
```
Comment (feanil) [inline playbooks/roles/veda_ffmpeg/defaults/main.yml:16]: Why this change?

File: playbooks/roles/edxapp/tasks/deploy.yml (hunk 1 above)
Comment (muhammad-ammar) [reply playbooks/roles/edxapp/tasks/deploy.yml]: This creates a [video pipeline integration record](https://studio-pipeline-ma777.sandbox.edx.org/admin/video_pipeline/videopipelineintegration/) in db so that edx-platform can communicate with veda to store transcript preferences for a course. This integration [record](https://courses.edx.org/admin/video_pipeline/videopipelineintegration/) is already created in production.

File: playbooks/roles/veda_ffmpeg/defaults/main.yml (hunk 2 above)
Comment (muhammad-ammar) [reply playbooks/roles/veda_ffmpeg/defaults/main.yml:16]: after some thought I decided that `/var/tmp/` is much better place to clone.

File: playbooks/roles/edxapp/tasks/deploy.yml (hunk 1 above)
Comment (feanil) [reply playbooks/roles/edxapp/tasks/deploy.yml]: I'm concernd about this running on all edxapp builds, can we do something similar to what we do for oauth codes and just put it on the sandboxes via a separate playbook at the end?

https://github.com/edx/configuration/blob/master/util/jenkins/ansible-provision.sh#L409-L420

**Review by feanil — APPROVED** [review]
